cfme/containers/provider/__init__.py

```diff
@@ -515,7 +516,7 @@ def is_displayed(self):
         return (
             self.title.is_displayed and
             self.breadcrumb.is_displayed and
-            self.breadcrumb.active_location == '{} (Summary)'.format(self.context['object'].name)
+            '{} (Summary)'.format(self.context['object'].name) in self.breadcrumb.active_location
```

Comment:
__QUESTION__
Why is this `in` instead of `==`?

Reply:
Since the navigation to image details change , and it use search on get entity, 
The active_loaction has additional text in the title, and it looks like for example:
dotnet/dotnetcore-10-rhel7 (Summary) (Names with "dotnet/dotnetcore-10-rhel7")
instead of only:
dotnet/dotnetcore-10-rhel7 (Summary)

In that case using "in" covers both the cases